docs/lasagne.rst

```diff
@@ -54,9 +54,12 @@ API
 
 .. automodule:: nolearn.lasagne
 
-  .. autoclass:: NeuralNet
+  .. autoclass:: NeuralNet(self,layers,**kwargs)
```

Comment:
Thanks for explaining.  Would you mind putting a space after the commas as is suggested by PEP8?